src/test/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheckTest.java

```diff
@@ -22,11 +22,6 @@
 import static com.puppycrawl.tools.checkstyle.checks.UniquePropertiesCheck.MSG_IO_EXCEPTION_KEY;
 import static com.puppycrawl.tools.checkstyle.checks.UniquePropertiesCheck.MSG_KEY;
 import static org.junit.Assert.assertEquals;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.times;
-import static org.powermock.api.mockito.PowerMockito.doNothing;
-import static org.powermock.api.mockito.PowerMockito.mockStatic;
-import static org.powermock.api.mockito.PowerMockito.verifyStatic;
```

Comment:
Can't `@RunWith(PowerMockRunner.class) @PrepareForTest(Closeables.class)` be removed from this test too? I don't see any other mocks.